Game/buildings.gd

```diff
@@ -0,0 +1,21 @@
+extends Node
+
+@export var _buildings_dict: Dictionary[Types.Building, BuildingResource]
+
+func get_icon(building_type: Types.Building) -> AtlasTexture:
+	if building_type != Types.Building.NONE:
+		return _buildings_dict[building_type].icon
+	else:
+		return null
+
+func get_atlas_coords(building_type: Types.Building) -> Vector2i:
+	if building_type != Types.Building.NONE:
+		return _buildings_dict[building_type].atlas_coordinates
+	else:
+		return Vector2i(-1, -1)
+
+func get_building_name(building_type: Types.Building) -> String:
```

Comment:
I have a similar criticism to above regarding the icon function, but I think that strings are potentially an interesting use case since you may not want to ever receive null. Though I'd probably prefer a "safely get string" method to this.